src/app/core/services/group.service.ts

```diff
@@ -0,0 +1,169 @@
+      operation?: string;
+      members: [string];
+    }
+
+    const body: NetCreate = {
+      title: create.title,
+      description: create.description,
+      operation: create.operation,
+      members: create.members,
+    };
+    return this.netService.postJSON<NetCreated>('/groups', body, {}).pipe(
+      map((res:NetCreated): Group => ({
+        id: res.id,
+        title: res.title,
+        description: res.description,
+        operation: res.operation,
+        members: res.members,
+      })),
+    );
+  }
+
+  /**
+   * Updates the given group, identified by its {@link Group.id}.
+   * @param update The updated Group.
+   */
+  updateGroup(update: Group): Observable<void> {
+    interface NetUpdate {
+      id: string;
+      title: string;
+      description: string;
+      operation?: string;
+      members: [string];
+    }
+
+    const body: NetUpdate = {
+      id: update.id,
+      title: update.title,
+      description: update.description,
+      operation: update.operation,
+      members: update.members,
+    };
+    return this.netService.putJSON(urlJoin('/groups', update.id), body, {});
+  }
+
+  /**
+   * Deletes the given group.
+   * @param groupId Id of the group to be deleted.
+   */
+  deleteGroup(groupId: string): Observable<void>{
+    return this.netService.delete(urlJoin('/groups', groupId),{});
+  }
+
+  /**
+   * Retrieves a Group with the given id.
+   * @param groupId The id of the group to be retrieved.
+   */
+  getGroupById(groupId: string): Observable<Group>{
+    interface NetGroup {
+      id: string;
+      title: string;
+      description: string;
+      operation?: string;
+      members: [string];
+    }
+
+    return this.netService.get<NetGroup>(urlJoin('/groups', groupId), {}).pipe(
+      map((res:NetGroup):Group => ({
+        id: res.id,
+        title: res.title,
+        description: res.description,
+        operation: res.operation,
+        members: res.members,
+      })),
+    );
+  }
+
+  /**
+   * Retrieves paginated groups fitting the given params.
+   * @param params Params for pagination.
+   */
+  getGroups(params: PaginationParams<GroupParams>): Observable<Paginated<Group>> {
+    interface NetEntry {
+      id: string;
+      title: string;
+      description: string;
+      operation?: string;
+      members: [string];
+    }
+
+    const nParams = netPaginationParams(params,(by:GroupParams)=> {
+      switch (by) {
+        case by.orderBy == GroupSort.ByTitle:
+          return 'title';
+        case by.orderBy == GroupSort.ByDescription:
+          return  'description';
+        case by.userId:
+          return 'by_user=' + by.userId;
+        case by.excludeGlobal:
+            return by.excludeGlobal? 'exclude_global=true' : 'exclude_global=false';
+        case  by.forOperation:
+          return 'for_operation=' + by.forOperation;
+        default:
+          throw new MDSError(MDSErrorCode.AppError, 'unknown group sort', {by: by})
+      }
+    });
```

Comment:
Bug and bad code formatting. Please read the documentation. The passed mapping function maps order-by-fields to public ones.
Set filters accordingly on the `nParams` object. Better: Declare an interface for acceptable params.